testing/file_test/file_test_base_test.cpp

```diff
@@ -31,7 +31,8 @@ class FileTestBaseTest : public FileTestBase {
     }
     stdout << "\n";
 
-    auto filename = path().filename();
+    auto filename =
+        std::filesystem::path(test_name().str()).filename().string();
```

Comment:
Why is the `.string()` call added here?

Reply:
Incorrectly assumed it was required; removed